.github/workflows/main.yaml

```diff
@@ -0,0 +1,129 @@
+name: main
+on:
+  pull_request:
+  push:
+    branches:
+      - 'main'
+env:
+  go-version: 1.15
+jobs:
+  build:
+    name: Build CKE
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.go-version }}
+      - name: Check static resources
+        run: |
+            cp ./static/resources.go /tmp/resources.go
+            make static
+            cmp /tmp/resources.go ./static/resources.go
+      - run: docker run -d --network host quay.io/coreos/etcd:v3.3
+      - run: make test-tools
+      - run: make test GOBIN=$(pwd)/docker
+      - name: build-image
+        run: docker build -t quay.io/cybozu/cke:latest ./docker
+  mtest:
+    name: Mtest
+    runs-on: ubuntu-20.04
+    needs: build
+    strategy:
+      matrix:
+        suite: [functions, robustness, operators]
+    env:
+      INSTANCE_NAME: cke-${{ matrix.suite }}-${{ github.run_number }}
+      SUITE: ${{ matrix.suite }}
+      CLUSTER: "cke-cluster.yml"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.go-version }}
+      - name: Set up Cloud SDK for cke
+        uses: google-github-actions/setup-gcloud@master
```

Comment:
master branch may be instable. I recommend to use the latest release (v0.2.1)

Reply:
Thank you. I'll fix it as you suggested.